src/System.Management.Automation/System.Management.Automation.csproj

```diff
@@ -7,7 +7,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\Microsoft.PowerShell.CoreCLR.AssemblyLoadContext\Microsoft.PowerShell.CoreCLR.AssemblyLoadContext.csproj" />
```

Comment:
Need to remove the entry from Powershell-Win.sln as well.

Reply:
Good point!

Reply:
Updated `PowerShell-Win.sln` and also deleted the file `Microsoft.PowerShell.CoreCLR.AssemblyLoadContext.csproj` (it was revived when I resolve conflicts).